.vscode/settings.json

```diff
@@ -1,7 +1,6 @@
 {
 	"files.exclude": {
 		"**/.hemttout": true,
-		"**/include": true,
```

Comment:
also I know this is unrelated, but personally I don't like this

hides it from search and from the file explorer, which is annoying